.github/workflows/review.yml

```diff
@@ -83,3 +83,33 @@ jobs:
           header: deployment-url
           message: |
             **Harrastus-helsinki branch is deployed to: ${{ env.CUSTOM_ENVIRONMENT_URL }}** :rocket::rocket::rocket:
+
+      # browser tests
+      - name: Setup Node
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: 14.x
+      - name: Create yarn cache directory path
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Install dependencies
+        run: yarn --prefer-offline --frozen-lockfile --check-files
+
+      - name: Run Acceptance Tests
+        id: testcafe
+        run: yarn test:browser:ci
+        env:
+          BROWSER_TESTS_ENV_URL: ${{ env.CUSTOM_ENVIRONMENT_URL }}
+      - name: Upload screenshots and videos of failed tests to artifact
```

Comment:
In yjdh browser test result is posted as pr comment, I think it's would be nice addition:
https://github.com/City-of-Helsinki/yjdh/blob/develop/.github/workflows/ks-review.yml#L291

Reply:
added